ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -151,6 +152,21 @@ Rectangle {
             }
         }
 
+        function copyMentions(start, end) {
+            copiedMentionsPos = []
+            for (var k = 0; k < mentionsPos.length; k++) {
+                if (mentionsPos[k].leftIndex >= start && mentionsPos[k].rightIndex <= end) {
+                    const mention = {
+                        name: mentionsPos[k].name,
+                        pubKey: mentionsPos[k].pubKey,
+                        leftIndex: mentionsPos[k].leftIndex - start,
+                        rightIndex: mentionsPos[k].rightIndex - start
+                    }
+                    copiedMentionsPos.push(mention)
+                }
```

Comment:
Sigh.... this component is getting slower and slower with more JS code :/ Not your fault I know but there has to be a limit, this file is over 1500 lines of code already